### ISSUE TITLE
+ character in login not supported ?

### ISSUE THREAD

**Issue by nsteinmetz** [scaleway/scaleway-cli#427]
Hi,

Using 1.9, I have:

```
nsteinmetz@swiip:~$ scw login
Login (cloud.scaleway.com): public+scaleway@mydomain.com
Password: 
FATA[0011] cannot execute 'login': Unable to contact ScalewayAPI: Validation Error 
```

So either the ``+`` is the issue or ``!`` is not supported in passwords or I need first to be part of the developer program but the pre-requisites does not seem to be mentionned somewhere (or I missed it)

Could you please tell me where the issue is ?

Thanks,
Nicolas

**Comment by QuentinPerez** [comment]
Hi @nsteinmetz 

Can you install the last version (1.11) https://github.com/scaleway/scaleway-cli/releases/tag/v1.11, it will resolve your issue
see https://github.com/scaleway/scaleway-cli/issues/418

**Comment by nsteinmetz** [comment]
Indeed fixed with v1.11 - thanks !
